ansible_collections/arista/avd/roles/eos_snapshot/templates/fabric_json_output.j2

```diff
@@ -1,3 +1,4 @@
+{# j2lint: disable=jinja-statements-delimeter #}
```

Comment:
This template should be refactored. Maybe just setting the all data into a temp variable and printing that with `to_nice_json`, instead of combining json with manual logic.